translations/de/info/task_description.html

```diff
@@ -343,40 +335,40 @@
 </table>
 
 <p>
-    You should help Stephan write a function to calculate this "multiplication".
-    You are given two positive integers (n&nbsp;>&nbsp;0), be careful with order of arguments.
+    Du sollst Stephan helfen eine Funktion zu schreiben, die diese Art von "Multiplikation"
+    anwenden kann.
+    Gegeben werden zwei positive Ganzzahlen (n&nbsp;>&nbsp;0).<br>
+    <strong>Achte auf die Reihenfolge der Argumente.</strong>
 </p>
 
 <p>
-    <strong>Input: </strong> Two arguments as integers.
+    <strong>Eingabe: </strong> Zwei Argumente als Ganzzahlen.
 </p>
 
 <p>
-    <strong>Output: </strong> The result of the Stephan's "multiplication", an integer.
+    <strong>Ausgabe: </strong> Das Ergebnis der "Stephan-Multiplikation", als ganze Zahl.
 </p>
 
-
 <div class="for_info_only">
-
     <p>
-        <strong>Example:</strong>
+        <strong>Beispiele:</strong>
     </p>
     <pre class="brush: python">
-checkio(4, 6) == 38
-checkio(2, 7) == 28
-checkio(7, 2) == 18
+        checkio(4, 6) == 38
+        checkio(2, 7) == 28
+        checkio(7, 2) == 18
     </pre>
 </div>
 
 <p class="for_info_only">
-    <strong>How it is used: </strong>
-    In this task we play around with logical binary operations, the basis for computer science.
-    Maybe you can find a use for this subject in cryptography?
-
+    <strong>Wozu das gut ist:</strong><br>
+    In dieser Aufgabe spielen wir mit logischen binären Operationen - der Grundlage der
+    Informatik - rum.<br>
```

Comment:
That ending, please not. End after Informatik. That's so to emphasize, you could even have an exclamation mark there.